packages/utils/src/__tests__/object.test.js

```diff
@@ -31,49 +29,50 @@ const object3 = {
 const assertSortingCorrectness = (sortingMethod, input, expectedValue) => {
   const arrayToSort = [...input];
   arrayToSort.sort(sortingMethod);
-  expect(arrayToSort).to.deep.equal(expectedValue);
+  expect(arrayToSort).toEqual(expectedValue);
```

Comment:
Although `toEqual` is possibly fine for what we want to achieve here, let's use the stricter version ([toStrictEqual](https://jestjs.io/docs/en/expect#tostrictequalvalue)]), unless if we have specific reasons to use `toEqual`

Similarly for all other cases in this PR

```suggestion
  expect(arrayToSort).toStrictEqual(expectedValue);
```